lib/iris/__init__.py

```diff
@@ -122,7 +122,7 @@ def callback(cube, field, filename):
 
 
 # Iris revision.
-__version__ = '2.0.0-DEV'
+__version__ = '2.0.dev0'
```

Comment:
To be fair, this doesn't belong in this PR, so I'll pull it ... but the `__version__` should be `PEP440`.

I'll save that for another day and another PR :+1: